app/Providers/Filament/AdminPanelProvider.php

```diff
@@ -49,6 +49,8 @@ public function panel(Panel $panel): Panel
                 'primary' => Color::hex('#2bb8b3'),
             ])
             ->viteTheme('resources/css/filament/admin/theme.css')
+            ->readOnlyRelationManagersOnResourceViewPagesByDefault(false)
+            // ->maxContentWidth('full')
```

Comment:
Let's delete this comment if we are not going to use it here.

Reply:
Leaving it pending ticket.